src/main/java/duke/Event.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a <code>Event</code> type of task.
+ */
+public class Event extends Task {
+    private final LocalDate date;
+
+    /**
+     * Constructs an instance of <code>Event</code>
+     * @param body Body message
+     * @param date Date of task
+     */
+    Event(String body, LocalDate date) {
+        super(body, false);
+        this.date = date;
+    }
+
+    /**
+     * Constructs an instance of <code>Event</code>
+     * @param body Body message
+     * @param done Status of task
+     * @param date Date of task
+     */
+    Event(String body, boolean done, LocalDate date) {
+        super(body, done);
+        this.date = date;
+    }
+
+    /**
+     * Returns date of <code>Event</code>
+     * @return Date
```

Comment:
Very small thing but maybe can be a little more specific in the javadocs (e.g. returns the data that event is happening)